src/components/Avatar.js

```diff
@@ -23,6 +24,9 @@ const propTypes = {
 
     /** The fill color for the icon. Can be hex, rgb, rgba, or valid react-native named color such as 'red' or 'blue' */
     fill: PropTypes.string,
+
+    /** Flag to check if avatar is from workspace, so that we can use fallback avatar accordingly */
+    fallbackIcon: PropTypes.string,
```

Comment:
propType should be `PropTypes.func`

Can you update the comment too it doesn’t make any sense.